packages/react/src/components/ExpandCollapsePanel/PanelTrigger.tsx

```diff
@@ -11,45 +11,61 @@ export interface PanelTriggerProps
   onClick?: (e: React.MouseEvent<HTMLButtonElement>) => void;
   iconExpanded?: IconType;
   iconCollapsed?: IconType;
+  shouldHideIcon?: boolean;
+  buttonRef?: Ref<HTMLButtonElement>;
 }
 
-function PanelTrigger({
-  children,
-  className,
-  open,
-  fullWidth,
-  onClick,
-  iconExpanded = 'chevron-down',
-  iconCollapsed = 'chevron-right',
-  ...other
-}: PanelTriggerProps) {
-  return (
-    <button
-      {...other}
-      className={classnames(
-        'ExpandCollapse__trigger',
-        fullWidth ? 'fullWidth' : '',
-        className
-      )}
-      type="button"
-      aria-expanded={open}
-      onClick={onClick}
-    >
-      <div className="ExpandCollapse__trigger-title">
-        {typeof children === 'function' ? children({ open: !!open }) : children}
-      </div>
-      <Icon type={open ? iconExpanded : iconCollapsed} />
-    </button>
-  );
-}
+const PanelTrigger = forwardRef<HTMLButtonElement, PanelTriggerProps>(
+  (
+    {
+      children,
+      className,
+      open,
+      fullWidth,
+      onClick,
+      buttonRef,
+      iconExpanded = 'chevron-down',
+      iconCollapsed = 'chevron-right',
+      shouldHideIcon = false,
+      ...props
+    }: PanelTriggerProps,
+    ref
+  ) => {
+    return (
+      <button
+        className={classnames(
+          className ? className : 'ExpandCollapse__trigger',
+          fullWidth ? 'fullWidth' : '',
+          open ? 'expanded' : ''
+        )}
+        type="button"
+        aria-expanded={open}
+        onClick={onClick}
+        ref={buttonRef}
+        {...props}
+      >
+        <div className="ExpandCollapse__trigger-title">
+          {typeof children === 'function'
+            ? children({ open: !!open })
+            : children}
+        </div>
+        {shouldHideIcon ? (
+          ''
+        ) : (
+          <Icon type={open ? iconExpanded : iconCollapsed} />
+        )}
+      </button>
+    );
+  }
+);
 
 PanelTrigger.propTypes = {
   children: PropTypes.oneOfType([PropTypes.func, PropTypes.node]),
   open: PropTypes.bool,
   onClick: PropTypes.func,
-  className: PropTypes.string,
-  iconExpanded: PropTypes.string,
-  iconCollapsed: PropTypes.string
```

Comment:
These prop types are causing a TypeScript issue. Not sure anything was changed that should have effected these. 

see: https://app.circleci.com/pipelines/github/dequelabs/cauldron/1429/workflows/2ec9949c-ab27-4a33-b535-1edf6531356e/jobs/6202